### PR TITLE
Feature/bug fix

### LINKED ISSUE #2142: OpenTracingSQLExecutionHook.toStringList NullPointerException
## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/sharding-sphere/sharding-sphere/issues).
- Read documentation: [ShardingSphere Doc](http://shardingsphere.io/document/current/en/overview/).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response **more than 7 days** and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ShardingSphere did you use?
3.0.1
### Which project did you use? Sharding-JDBC or Sharding-Proxy?
Sharding-JDBC
### Expected behavior
sharding-opentracing log trace without effect main execute
### Actual behavior
log trace throw NullPointerException effect sql execute
### Reason analyze (If you can)
OpenTracingSQLExecutionHook.toStringList method, when params has null, such as insert, call guava Joiner.toString throw NullPointerException 
### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
As above
### Example codes for reproduce this issue (such as a github link).
As above

### PR DESCRIPTION
**Pull request by Moriadry**
Fixes #2142.

Changes proposed in this pull request:
- fix null pointer exception when we pass null as parameters in SQLUnit
- add unit test to esure it.
-

### CODE REVIEW

**Comment by Moriadry** [comment]
hi @tuohai666 ! could u help me review this bug fix :)

File: sharding-opentracing/src/main/java/org/apache/shardingsphere/opentracing/hook/OpenTracingSQLExecutionHook.java [hunk 1]
```diff
@@ -61,7 +61,11 @@ public void start(final RouteUnit routeUnit, final DataSourceMetaData dataSource
     }
     
     private String toString(final List<Object> parameterSets) {
-        return parameterSets.isEmpty() ? "" : String.format("[%s]", Joiner.on(", ").join(parameterSets));
+
```
Comment (tuohai666) [inline sharding-opentracing/src/main/java/org/apache/shardingsphere/opentracing/hook/OpenTracingSQLExecutionHook.java]: remove the empty line

File: sharding-opentracing/src/test/java/org/apache/shardingsphere/opentracing/hook/OpenTracingSQLExecutionHookTest.java [hunk 2]
```diff
@@ -95,6 +95,30 @@ public void assertExecuteSuccessForTrunkThread() {
         assertThat(actualTags.get(Tags.DB_STATEMENT.getKey()), CoreMatchers.<Object>is("SELECT * FROM success_tbl;"));
         assertThat(actualTags.get(ShardingTags.DB_BIND_VARIABLES.getKey()), CoreMatchers.<Object>is("[1, 2]"));
         verify(activeSpan, times(0)).deactivate();
+
```
Comment (tuohai666) [inline sharding-opentracing/src/test/java/org/apache/shardingsphere/opentracing/hook/OpenTracingSQLExecutionHookTest.java]: remove the empty line

**Comment by Moriadry** [comment]
Already fix it.

**Comment by tuohai666** [comment]
@Moriadry 
Accept. Thanks for your effort.

**Comment by Moriadry** [comment]
@tuohai666  one more thing, I'm trying to help with some sharding-proxy bug like this: https://github.com/apache/incubator-shardingsphere/issues/2235 . But I don't know how to run this case locally, could give me some advice? Many thanks!

**Comment by tuohai666** [comment]
Let's talk in that issue instead of this place.

**Pull request merged**